src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsEngine.java

```diff
@@ -466,6 +466,12 @@ public VariantContextWriter setupVCFWriter(Set<VCFHeaderLine> defaultToolVCFHead
                 attrs.put(GATKVCFConstants.HAPLOTYPE_CALLER_PHASING_GT_KEY, GenotypeGVCFs.PHASED_HOM_VAR_STRING);
             }
 
+            //update PGT for hom refs
+            if ( oldGT.isHomRef() && oldGT.hasExtendedAttribute(GATKVCFConstants.HAPLOTYPE_CALLER_PHASING_GT_KEY) ) {
+                attrs.remove(GATKVCFConstants.HAPLOTYPE_CALLER_PHASING_GT_KEY);
+                attrs.remove(GATKVCFConstants.HAPLOTYPE_CALLER_PHASING_ID_KEY);
```

Comment:
Should we also be removing PS here as well?  Don't we output those now?  VCFConstants.PHASE_SET_KEY

Reply:
We should probably also remove if it's homvar right?

Reply:
HomVars get converted to 1|1 above, which isn't super helpful, but it's not wrong.